frontend/src/views/MemberVotePage.vue

```diff
@@ -17,69 +17,36 @@
         </b-col>
         <b-col cols="auto">
           <session-leave-button />
-          <estimate-timer
```

Comment:
why reformatting?

Reply:
Sorry, didnt notice. My IDE did this on saving. My bad. 😯